install

```diff
@@ -2,20 +2,26 @@
 
 cd `dirname $0`
 
-# The clean script removes temporary and junk files that should not be
-# committed to the source code repo.
-if (test -e clean)
-then
-    ./clean
-fi
-
 if [ "$1" = "--skip-types" ]; then
     shift;
     dont_skip_types=0
 else
     dont_skip_types=1
 fi
 
+dev_install=0
+if [ "$1" == "--dev" ]; then
```

Comment:
This doesn't work if the script is called with the flags in order `--dev --skip-types`?